base/libgit2/repository.jl

```diff
@@ -183,7 +183,7 @@ function reset!(repo::GitRepo, obj::GitObject, mode::Cint;
 end
 
 function clone(repo_url::AbstractString, repo_path::AbstractString,
-               clone_opts::CloneOptions = CloneOptions())
+               clone_opts::CloneOptions)
```

Comment:
It's in libgit2.jl, where there is a 2-argument `clone` method that collides due to this argument being optional.